packages/loot-core/src/server/api.test.ts

```
@@ -0,0 +1,48 @@
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+
+    mockServerHandlers = {
+      'accounts-bank-sync': jest.fn().mockResolvedValue({ errors: [] }),
+    };
+
+    // Remove the accounts-bank-sync handler if it exists
+    // or it won't be replaced by the mock
+    if (handlers) {
+      delete handlers['accounts-bank-sync'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)